gce_eligibility_hexes.sql

```diff
@@ -0,0 +1,290 @@
+CREATE MATERIALIZED VIEW gce_eligibility_hexes AS (
+    WITH map_bounds AS (
+        SELECT
+            MIN(ST_Y(geom)) as min_latitude,
+            MAX(ST_Y(geom)) as max_latitude,
+            MIN(ST_X(geom)) as min_longitude,
+            MAX(ST_X(geom)) as max_longitude
+        FROM
+            gce_eligibility
+    ),
+    zoom_level_hexes AS (
+        SELECT
+            10 as zoom_level,
+            geom
+        FROM
+            ST_HexagonGrid(
+                0.005,
```

Comment:
these hex sizes are pretty arbitrary, I was hoping to dial them in with some trial and error

Reply:
honestly, they look pretty good to me. I'm sure they could be a little better but I'm happy with it as-is for now.